example/magnet/magnet.cpp

```diff
@@ -9,6 +9,7 @@
 #include <boost/url/optional.hpp>
 #include <boost/url/rfc/absolute_uri_rule.hpp>
 #include <boost/url/grammar/digit_chars.hpp>
+#include <boost/url/grammar/parse.hpp>
```

Comment:
I included this in a separate commit in passing to avoid the CI errors. 

This is already fixed in passing in another PR, but this one is a simpler and a higher priority one.